src/components/pages/dashboard-pages/Students.jsx

```diff
@@ -4,6 +4,7 @@ import {
   getAllUsersInfo,
```

Comment:
This component is working just fine. So let's not worry about refactoring. However, there is a lot going on. For future codebases just keep in mind to keep concerns separated. Also, a name like `TeacherStudent` is ambiguous so it makes it even more difficult as a developer to decide what logic belongs inside.

Reply:
Thank you for the feedback! I’ll keep this in mind for future work and will aim to better organize the logic and choose more descriptive names for components.